### DISCUSSION TITLE
Patreon integration

### DISCUSSION THREAD

**Discussion by hassan-sd** [civitai/civitai#171] [Ideas]
As a Patreon page owner, I'd like the ability to have my patron members connect their patron details to Civitai, allowing me as a page owner to provide content through Civitai directly to specific tiers of members.

ie:

- As a patreon page owner, I can release content to patron users who can access the model via their Civitai + Patron integrated account
 _The download URL must be auth based and not allow general public access_
 _I can distinguish patron users vs non patron users on model reviews/feedback/comments_
 
 - As a patron user, I can connect my patron account to my Civitai account, sharing my subscription tier details with Civitai in order to gain access to content released from a patron page through Civitai
 - As a patron user, I can access content that is _only_ available to patrons subbed to a specific page who is present on Civitai

![image](https://user-images.githubusercontent.com/119671806/214038436-fdaf149b-346d-4b77-85ac-d4b0946cb83c.png)

![image](https://user-images.githubusercontent.com/119671806/214039096-1b64a35c-7ef7-4e4f-8085-e69df1bfd3c2.png)

**Comment by Trahloc** [comment]
Isn't this having civitai act as a free extension to Patreon? Makes sense to have a link back for folks like yourself creating models but having civitai host restricted content that their own supporters can't access ... doesn't seem like the benefit goes both ways there.
